### PR TITLE
Remove --container-runtime kubelet flag for 1.27+

### PR DESCRIPTION
**Pull request by cartermckinnon**
**Description of changes:**

This flag is gone in 1.27, so it's removed in the default `systemd` service unit, and conditionally added back for 1.26-.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

**Testing Done**

TODO

### CODE REVIEW

File: files/bootstrap.sh [hunk 1]
```diff
@@ -191,6 +191,8 @@ if vercmp "$KUBELET_VERSION" lt "1.27.0"; then
 fi
 MOUNT_BPF_FS="${MOUNT_BPF_FS:-$DEFAULT_MOUNT_BPF_FS}"
 
+KUBELET_ARGS=""
```
Comment (mmerkes) [inline files/bootstrap.sh:194]: Why is this needed?

Comment (cartermckinnon) [reply files/bootstrap.sh:194]: I was misreading line numbers while jumping around this file -- it's not needed, for now.
